third_party/rust/crates/crates.bzl

```diff
@@ -257,6 +258,15 @@ def raze_fetch_remote_crates():
         build_file = Label("//third_party/rust/crates/remote:BUILD.aho-corasick-0.7.18.bazel"),
     )
 
+    maybe(
+        http_archive,
```

Comment:
Forgot to mention this, I think these need a `sha256` entry.

I patched the PR in locally and built opentitantool, and I saw these warnings:

```
DEBUG: Rule 'raze__chrono__0_4_22' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "bfd4d1b31faaa3a89d7934dbded3111da0d2ef28e3ebccdb4f0179f5929d1ef1"
DEBUG: Repository raze__chrono__0_4_22 instantiated at:
  /usr/local/google/home/dmcardle/REPOS/opentitan/WORKSPACE:50:10: in <toplevel>
  /usr/local/google/home/dmcardle/REPOS/opentitan/third_party/rust/deps.bzl:28:29: in rust_deps
  /usr/local/google/home/dmcardle/REPOS/opentitan/third_party/rust/crates/crates.bzl:409:10: in raze_fetch_remote_crates
  /usr/local/google/home/dmcardle/.cache/bazel/_bazel_dmcardle/b5c6bad67a60fe0f0e2dfcf06963df7b/external/bazel_tools/tools/build_defs/repo/utils.bzl:233:18: in maybe
Repository rule http_archive defined at:
  /usr/local/google/home/dmcardle/.cache/bazel/_bazel_dmcardle/b5c6bad67a60fe0f0e2dfcf06963df7b/external/bazel_tools/tools/build_defs/repo/http.bzl:353:31: in <toplevel>
DEBUG: Rule 'raze__time__0_1_44' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "6db9e6914ab8b1ae1c260a4ae7a49b6c5611b40328a735b21862567685e73255"
DEBUG: Repository raze__time__0_1_44 instantiated at:
  /usr/local/google/home/dmcardle/REPOS/opentitan/WORKSPACE:50:10: in <toplevel>
  /usr/local/google/home/dmcardle/REPOS/opentitan/third_party/rust/deps.bzl:28:29: in rust_deps
  /usr/local/google/home/dmcardle/REPOS/opentitan/third_party/rust/crates/crates.bzl:1550:10: in raze_fetch_remote_crates
  /usr/local/google/home/dmcardle/.cache/bazel/_bazel_dmcardle/b5c6bad67a60fe0f0e2dfcf06963df7b/external/bazel_tools/tools/build_defs/repo/utils.bzl:233:18: in maybe
Repository rule http_archive defined at:
  /usr/local/google/home/dmcardle/.cache/bazel/_bazel_dmcardle/b5c6bad67a60fe0f0e2dfcf06963df7b/external/bazel_tools/tools/build_defs/repo/http.bzl:353:31: in <toplevel>
DEBUG: Rule 'raze__iana_time_zone__0_1_47' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "4c495f162af0bf17656d0014a0eded5f3cd2f365fdd204548c2869db89359dc7"
DEBUG: Repository raze__iana_time_zone__0_1_47 instantiated at:
  /usr/local/google/home/dmcardle/REPOS/opentitan/WORKSPACE:50:10: in <toplevel>
  /usr/local/google/home/dmcardle/REPOS/opentitan/third_party/rust/deps.bzl:28:29: in rust_deps
  /usr/local/google/home/dmcardle/REPOS/opentitan/third_party/rust/crates/crates.bzl:707:10: in raze_fetch_remote_crates
  /usr/local/google/home/dmcardle/.cache/bazel/_bazel_dmcardle/b5c6bad67a60fe0f0e2dfcf06963df7b/external/bazel_tools/tools/build_defs/repo/utils.bzl:233:18: in maybe
Repository rule http_archive defined at:
  /usr/local/google/home/dmcardle/.cache/bazel/_bazel_dmcardle/b5c6bad67a60fe0f0e2dfcf06963df7b/external/bazel_tools/tools/build_defs/repo/http.bzl:353:31: in <toplevel>
DEBUG: Rule 'raze__once_cell__1_14_0' indicated that a canonical reproducible form can be obtained by modifying arguments sha256 = "2f7254b99e31cad77da24b08ebf628882739a608578bb1bcdfc1f9c21260d7c0"
DEBUG: Repository raze__once_cell__1_14_0 instantiated at:
  /usr/local/google/home/dmcardle/REPOS/opentitan/WORKSPACE:50:10: in <toplevel>
  /usr/local/google/home/dmcardle/REPOS/opentitan/third_party/rust/deps.bzl:28:29: in rust_deps
  /usr/local/google/home/dmcardle/REPOS/opentitan/third_party/rust/crates/crates.bzl:1041:10: in raze_fetch_remote_crates
  /usr/local/google/home/dmcardle/.cache/bazel/_bazel_dmcardle/b5c6bad67a60fe0f0e2dfcf06963df7b/external/bazel_tools/tools/build_defs/repo/utils.bzl:233:18: in maybe
Repository rule http_archive defined at:
  /usr/local/google/home/dmcardle/.cache/bazel/_bazel_dmcardle/b5c6bad67a60fe0f0e2dfcf06963df7b/external/bazel_tools/tools/build_defs/repo/http.bzl:353:31: in <toplevel>
```

Reply:
This is fixed by @alphan's latest commit (`cargo raze --generate-lockfile`).

Reply:
Thanks for checking @dmcardle!